tryingsomethingyetagain/team2120/team2120/QuizViewController.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  QuizViewController.swift
+//  team2120
+//
+//  Created by Allison Fain on 10/6/22.
+//
+
+import UIKit
+
+class QuizViewController: UIViewController {
```

Comment:
Do we only need one quiz view controller? Are we certain we are only doing one quiz format?